src/components/App/App.js

```diff
@@ -1,18 +1,50 @@
 import './App.css';
 import banner from '../../images/old-school-theater-banner.png';
-import movieData from '../../data/data';
 import MoviesBox from '../MoviesBox/MoviesBox';
 import SingleMovie from '../SingleMovie/SingleMovie';
 import ExitMovie from '../ExitMovie/ExitMovie';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
+import { getAllMovies, getSingleMovie } from '../../apiCalls';
 
 function App() {
-  const [allMovies, setAllMovies] = useState(movieData.movies);
+  const [allMovies, setAllMovies] = useState([]);
   const [singleMovie, setSingleMovie] = useState(null);
```

Comment:
To minimize potential bugs in our code, it is advisable to avoid using null as the initial value for state whenever possible.